easybuild/easyconfigs/b/Bison/Bison-3.0.4-gimkl-2017a.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'Bison'
+version = '3.0.4'
+
+homepage = 'http://www.gnu.org/software/bison'
+description = """Bison is a general-purpose parser generator that converts an annotated context-free grammar
+ into a deterministic LR or generalized LR (GLR) parser employing LALR(1) parser tables."""
+
+toolchain = {'name': 'gimkl', 'version': '2017a'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
+builddependencies = [('M4', '1.4.17', '', True)]
```

Comment:
@valtandor We generally don't fix dependencies to a particular subtoolchain, that's a site-specific choice...